Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp

```diff
@@ -422,12 +422,12 @@ void WebKitProtocolHandler::handleGPU(WebKitURISchemeRequest* request)
         if (auto* displayLink = page.process().processPool().displayLinks().existingDisplayLinkForDisplay(*displayID)) {
             auto& vblankMonitor = displayLink->vblankMonitor();
             addTableRow(displayObject, "VBlank type"_s, vblankMonitor.type() == DisplayVBlankMonitor::Type::Timer ? "Timer"_s : "DRM"_s);
-            addTableRow(displayObject, "VBlank refresh rate"_s, makeString(vblankMonitor.refreshRate(), "Hz"));
+            addTableRow(displayObject, "VBlank refresh rate"_s, makeString(vblankMonitor.refreshRate(), "Hz"_s));
         }
     }
 
 #if USE(LIBDRM)
-    if (strcmp(policy, "never")) {
+    if (strcmp(policy.characters(), "never")) {
```

Comment:
Kind of wish we had an `!=` we could use here instead.

Reply:
Actually, I believe we do for ASCIILiteral. I'll update.